lib/cylc/task_proxy.py

```diff
@@ -312,7 +312,6 @@ def __init__(
             self.db_inserts_map[self.TABLE_TASK_STATES].append({
                 "time_created": get_current_time_string(),
                 "time_updated": get_current_time_string(),
-                "try_num": self.run_try_state.num + 1,
```

Comment:
I think the try number is still used. It is the number of automatic retries as opposed to the number of submits (manual or automatic).